q2_demux/plugin_setup.py

```diff
@@ -125,7 +125,9 @@
     function=q2_demux.summarize,
     inputs={'data':
             SampleData[SequencesWithQuality | PairedEndSequencesWithQuality]},
-    parameters={'n': qiime2.plugin.Int},
+    parameters={
```

Comment:
Is this stylistic change necessary? Is it going to confuse someone when they run `git blame`?